lib/Distribution/Opensuse/Leap/16Latest.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class represents current (i.e. latest) SLE 16 distribution and
+# provides access to its features.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Distribution::Opensuse::Leap::16Latest;
+use parent Distribution::Opensuse::AgamaDevel;
+use strict;
+use warnings FATAL => 'all';
+
+use Yam::Agama::Pom::GrubMenuLeapPage;
```

Comment:
I'm not sure if it is strictly neede or fails silently, but we also add:
`Yam::Agama::Pom::GrubMenuBasePage`

Reply:
updated